tests/surface/test_spheroid.py

```diff
@@ -262,8 +262,8 @@ def runTest(self):
             ref = t.d_dpos.vals[...,i]
             self.assertTrue(abs(dt_dpos/ref - 1).max() < 1.e-5)
 
-        Path.reset_registry()
-        Frame.reset_registry()
+#         Path.reset_registry()
+#         Frame.reset_registry()
```

Comment:
In this and other similar files, why are these commented out instead of deleted?